### PR TITLE
Add "type" entries

### PR DESCRIPTION
**Pull request by jameshale**
"Type" entries for enabled, openfield, closefield, exitfield and textchanged added

### CODE REVIEW

**Comment by jameshale** [comment]
I noticed these entries were appearing in my Dash docset under the type "Widget".
Examination of the lcb showed that the type spec for the rogue properties/messages were missing.
I made sep pull requests for each affected widget.
This also showed that fixing one omission can reveal another and that the Travis log files are very long.

File: extensions/widgets/androidfield/androidfield.lcb [hunk 1]
```diff
@@ -19,6 +19,9 @@ along with LiveCode.  If not see <http://www.gnu.org/licenses/>.  */
 This widget is a native field on Android.
 
 Name: enabled
+Type: property
+
+Summary: Whether as field is active and responds to user action
```
Comment (montegoulding) [inline extensions/widgets/androidfield/androidfield.lcb]: @jameshale `Whether the field`

**Comment by montegoulding** [comment]
@jameshale it looks like the other three PRs contain the changes in this one. I think we can deal with this in the one PR so they can be closed if they don't add anything.

**Comment by jameshale** [comment]
@montegoulding the other pr’s are for different files. I thought in this case they should be distinct. Does it matter?

**Comment by jameshale** [comment]
Oh, and I don’t know how to combine them at this stage anyway.

**Comment by livecodeali** [comment]
@jameshale No worries, what should be in one PR is not clearly defined. If you're doing a similar small thing in a bunch of files then one would usually suffice (and is slightly easier for us to deal with), but it's not a big issue. I believe all of your other PRs have the same `whether as` instead of `whether the` typo

**Comment by montegoulding** [comment]
Sorry @jameshale my bad... I didn't look at the PRs closely enough

**Comment by jameshale** [comment]
all done - dang the convenience of copy and paste.

**Comment by montegoulding** [comment]
@livecode-vulcan review ok 5e32ba0

**Comment by livecode-vulcan** [comment]
💙 review by @montegoulding ok 5e32ba004cf7b744dba5423c23cfddb6c366be29

**Comment by livecode-vulcan** [comment]
😎 test success 5e32ba004cf7b744dba5423c23cfddb6c366be29

* try-community-armv6-android-api8: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api8/builds/1140)
* try-community-armv6-android-api9: [success](https://vulcan.livecode.com/builders/try-community-armv6-android-api9/builds/1156)
* try-community-js-emscripten-sdk1.35: [success](https://vulcan.livecode.com/builders/try-community-js-emscripten-sdk1.35/builds/1130)
* try-community-universal-ios-iphoneos11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphoneos11.2/builds/83)
* try-community-universal-ios-iphonesimulator11.2: [success](https://vulcan.livecode.com/builders/try-community-universal-ios-iphonesimulator11.2/builds/83)
* try-community-universal-mac-macosx10.6: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.6/builds/1135)
* try-community-universal-mac-macosx10.9: [success](https://vulcan.livecode.com/builders/try-community-universal-mac-macosx10.9/builds/1157)
* try-community-x86-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian7/builds/1140)
* try-community-x86-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86-linux-debian8/builds/1156)
* try-community-x86_64-linux-debian7: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian7/builds/1139)
* try-community-x86_64-linux-debian8: [success](https://vulcan.livecode.com/builders/try-community-x86_64-linux-debian8/builds/1149)
* try-community-x86-win32: [success](https://vulcan.livecode.com/builders/try-community-x86-win32/builds/1173)
* try-community-x86_64-win32: [success](https://vulcan.livecode.com/builders/try-community-x86_64-win32/builds/1134)

**Pull request merged**